test/services/fileStorage/index.test.js

```diff
@@ -82,8 +87,10 @@ describe('fileStorage services', () => {
 			...fixtures.roles.map((_) => RoleModel.findByIdAndRemove(_._id).exec()),
 			...fixtures.courses.map((_) => courseModel.findByIdAndRemove(_._id).exec()),
 		];
-
-		return Promise.all(promises);
+		await server.close();
+		await Promise.all(promises);
```

Comment:
maybe close the server after the promises are through?
(this is going to cause some fun merge conflicts with my PR :) )

Reply:
oh yes.. XD